### ISSUE TITLE
Authlib 2

### ISSUE THREAD

**Issue by maximmasterr** [yushijinhun/authlib-injector#120]
Minecraft versions since 1.16.4 use authlib 2.x.x, and current version of authlib injector fails with `com.mojang.authlib.exceptions.AuthenticationUnavailableException: null` in minecraft 1.16.4 and 1.16.5
or 
```
[08:28:52] [Render thread/ERROR]: Failed to verify authentication
com.mojang.authlib.exceptions.InvalidCredentialsException: Status: 401
```
in minecraft 1.17
[logs](https://pastebin.com/QR4DA94j)

**Comment by yushijinhun** [comment]
The error in your log doesn't affect any functionality.

I have just confirmed that skins can't be displayed in Minecraft 1.17, which is caused by [a change in authlib 2.3.31](https://gitlab.com/yushijinhun/authlib-history/-/commit/3d6ddb5e2edf53ee3c3f83c10f6c2e0dc2441498#f2ddad9b65457fc7d5ccdfc873f6a9dfaf4ea922_211_216). I will fix it soon.

**Comment by yushijinhun** [comment]
Fixed in #124.
